Os/Posix/TaskRoot.cpp

```diff
@@ -41,34 +41,10 @@ namespace Os {
 
         I32 stat = pthread_attr_init(&att);
         if (stat != 0) {
-            Fw::Logger::logMsg("pthread_attr_init: (%d)(%d): %s\n",stat,errno,strerror(stat));
+            Fw::Logger::logMsg("pthread_attr_init: (%d)(%d): %s\n",stat,errno, reinterpret_cast<POINTER_CAST>(strerror(stat)));
         	return TASK_INVALID_PARAMS;
         }
 
-        stat = pthread_attr_setstacksize(&att,stackSize);
```

Comment:
Why were these lines removed?

Reply:
`pthread_attr_setstacksize(&att,stackSize)` always returned 22 for me -> Led to segfault. 
Same for other attributed functions. 
I practically followed the setup of task.cpp where these attributes are not set for the linux configuration. 
However, I did not look into further details here.

Reply:
This was why we had a separate file for running as root, since many of these functions don't work in userspace. Can we keep the same partition in the file and add the CPU affinity functions to the regular `Task.cpp`?